src/BinaryMapper.Windows.Minidump/MinidumpMapper.cs

```diff
@@ -51,9 +84,41 @@ public Minidump ReadMinidump(Stream stream)
                         break;
                     case MINIDUMP_STREAM_TYPE.MemoryListStream:
                         minidump.MemoryListStream = _streamBinaryMapper.ReadObject<MINIDUMP_MEMORY_LIST_STREAM>(stream);
+
+                        // Fill referenced data
```

Comment:
This will use up alot of memory, we are now loading all memory included in the dump into RAM.
This MemoryRanges list, together with the Stacks in the ThreadList take up considerable space.

We could make this loading optional, add a bool parameter for it? Or a property inside MinidumpMapper?
We could also completely remove it, and show the uesr how to do it manually after the fact.

Could just make a method
`void ReplaceLocationWithLocationWithData(ref MINIDUMP_LOCATION_DESCRIPTOR x)`
or
`MINIDUMP_LOCATION_DESCRIPTOR_WITH_DATA GetLocationData(MINIDUMP_LOCATION_DESCRIPTOR x)`